### PR TITLE
add kubernetes manifests + helm templates for explorer components

### PR DESCRIPTION
**Pull request by DaMandal0rian**
This PR add supports for deploying the block-explorer and squid-archive with kubernetes using plain manifest files and helm charts.

### CODE REVIEW

File: explorer/k8s/helm/archive-squid/templates/postgres-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    min_wal_size = 1GB
+    max_wal_size = 4GB
+    max_worker_processes = 16
+    max_parallel_workers_per_gather = 4
+    max_parallel_workers = 16
+    max_parallel_maintenance_workers = 4
+    shared_preload_libraries='pg_stat_statements'
+    pg_stat_statements.track=all
+    # Replace with particular IP addresses in production
+    listen_addresses = 'REPLACE_IP_ADDRESS'
```
Comment (isSerge) [inline explorer/k8s/helm/archive-squid/templates/postgres-configmap.yaml:41]: should be a variable?

File: explorer/k8s/helm/archive-squid/values.yaml [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+  repository: ghcr.io/subspace/blockarchive-ingest
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+image_node:
+  repository: ghcr.io/subspace/node
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "snapshot-DATE"
```
Comment (isSerge) [inline explorer/k8s/helm/archive-squid/values.yaml:23]: should be a real tag?

**Comment by isSerge** [comment]
maybe there is confusion about these names: `archive-squid` and `explorer-squid` - we have [Subsquid Archive](https://github.com/subspace/infra/tree/main/_docs/7_SubsquidArchive) and [Blockexplorer Squid](https://github.com/subspace/infra/tree/main/_docs/8_BlockExplorerSquid) or simply Archive and Squid

File: explorer/k8s/helm/archive-squid/templates/postgres-configmap.yaml (hunk 1 above)
Comment (DaMandal0rian) [reply explorer/k8s/helm/archive-squid/templates/postgres-configmap.yaml:41]: Good catch thanks 🙏

**Pull request closed without merging**